public/PyodideWorker.js

```diff
@@ -1,7 +1,7 @@
-/* global importScripts, loadPyodide, SharedArrayBuffer, Atomics */
+/* global importScripts, loadPyodide, SharedArrayBuffer, Atomics, pygal, _internal_sense_hat */
 
-import * as pygal from "./pygal.js";
-import * as _internal_sense_hat from "./_internal_sense_hat.js";
+importScripts("http://localhost:3011/_internal_sense_hat.js");
+importScripts("http://localhost:3011/pygal.js");
```

Comment:
💭 this will only work for local dev. I'm wondering if `importScripts` allows relative imports? Or if we can do something like
```
importScripts(`${PUBLIC_URL}/_internal_sense_hat.js`);
```
it might require some webpack env var parsing though in the webpack config so this is populated on build

Reply:
I _think_ this works but sorry for the small comment that threw up a couple of spanners 😆